WordPress/gutenberg
# Site Editor: Fix bug where focus moved erroneously in navigation screens.

Fixes #44191.



## What?

Site Editor: Fix bug where focus moved back erroneously to the Navigation Back Button after an input onChange event fired. Now focus remains with the focused input.

## Why?

Fixes a bug that interrupts user input, when focus moves incorrectly.

## How?

Added a check to see if the navigation screen already has a focused element within it, and prevent the item from getting focused

## Testing Instructions
1. Open Site Editor.
2. Open Global Styles Sidebar.
3. Open Layout Screen.
4. Back button should have focus.
5. Move focus into an input field and enter a value.
6. Verify that focus remained with the input field you moved into.

## Screenshots or screencast
